evaluation.md

```diff
@@ -5,6 +5,33 @@
 _Describe each change from your ideal syntax to the syntax you implemented, and
 describe_ why _you made the change._
 
+The first compromise we had to make was changing from "your nth weapon" to "weapon 1" because we didn't want to
+implement 99 different functions to represent the different ordinal numbers.
+
```

Comment:
we ran into a similar problem, not the same but basically just the problem of naming things so that your syntax can be at all nice